Improving the readme (#3265)
minvws/nl-covid19-data-dashboard: README.md

> # NL Corona Dashboard

Reviewer: ```suggestion
# NL Coronavirus Dashboard
```

This is what they use on the English language website as well.

The dashboard provides information on the outbreak and prevalence of COVID-19 in
The Netherlands. It combines measured and modelled data from various sources to
give a broad perspective on the subject.

## Disclaimer

This dashboard is developed and maintained by a different team than the NL
COVID-19 Notification App. They are separate projects. If you want to get in
touch with that team, please join the CODE for NL Slack and join the channel
`#coronadashboard`.

> Tamas Erkelens from the Municipality of Amsterdam is contact person for the

Reviewer 2: ```suggestion
Tamas Erkelens from the Municipality of Amsterdam is the contact person for the
```

project team that made the dashboard.

[CODE For NL Slack](https://doemee.codefor.nl)

## Development & Contribution process

The core team works directly from this open-source repository. If you plan to

> propose changes, we recommend to open an issue beforehand where we can discuss

Reviewer 2: ```suggestion
propose changes, we recommend opening an issue beforehand where we can discuss
```

your planned changes. This increases the chance that we might be able to use
your contribution (or it avoids doing work if there are reasons why we wouldn't
be able to use it).

## Packages

The project is set up as a monorepo and therefore the code is organized in
multiple packages.

> - `app`: The main application that contains front-end part of the dashboard.

Reviewer 2: ```suggestion
- `app`: The main application that contains the front-end part of the dashboard.
```
